packages/cmf/src/RegistryProvider.js

```diff
@@ -27,6 +35,35 @@ export default class RegistryProvider extends React.Component {
 	}
 }
 
+function Register(props, context) {
+	let id;
+	let item;
```

Comment:
Remove those declaration and declare `const` in the 1st `if` bloc

Reply:
my bad, was for a previous implementation, thanks 👍 
I will update that